qiskit/ignis/mitigation/measurement/fitters.py

```diff
@@ -211,27 +212,27 @@ def plot_calibration(self, ax=None, show_plot=True):
 
 class TensoredMeasFitter():
     """
-    Measurement correction fitter for a tensored calibration
+    Measurement correction fitter for a tensored calibration.
     """
 
     def __init__(self, results, mit_pattern,
```

Comment:
types are missing (in TensoredMeasFitter)

Reply:
done